clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/ConsumerExample.java

```diff
@@ -42,23 +45,25 @@ public static void main(String[] args) throws Exception {
     Properties props = loadConfig(args[0]);
 
     // Add additional properties.
-    props.put("enable.auto.commit", "true");
-    props.put("auto.commit.interval.ms", "1000");
-    props.put("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
-    props.put("value.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
+    props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, "org.apache.kafka.common.serialization.StringDeserializer");
+    props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, "io.confluent.kafka.serializers.KafkaJsonDeserializer");
+    props.put(KafkaJsonDeserializerConfig.JSON_VALUE_TYPE, DataRecord.class);
+    props.put(ConsumerConfig.GROUP_ID_CONFIG, "java_example_group_1");
+    props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
```

Comment:
@hachikuji I think the idea here is to show-case an example application. The application requires auto.offset.reset=earliest and the example group id (to avoid conflict with proper groups), so I thin kit is fine to override whatever was configured in .ccloud/config for these properties.

Reply:
@ybyzek Yeah, that's what I was referring to. I guess we are mainly using that config to provide connection-related configs.

@edenhill Yep, seems reasonable to me. My brain is programmed to notice these things, but it seems sensible for the example.